frontend/src/features/nextcloud_integration/components/FileSelectorModal.tsx

```diff
@@ -35,17 +40,16 @@ export default function FileSelectorModal({
         show={show}
       />
       <ModalContainer show={show}>
-        <div className="flex h-[70vh] w-[80vw] flex-col rounded-lg bg-neutral-100 p-6 dark:bg-neutral-100-dark">
-          <div className="flex justify-between">
-            {/* <h2>{title}</h2> */}
-            <button
-              onClick={onCancel}
-              className="rounded-lg border border-neutral-600 px-5 py-2.5 text-center text-sm font-medium hover:bg-neutral-600 focus:outline-none focus:ring-4 focus:ring-blue-300 sm:w-auto"
-            >
-              x
-            </button>
-          </div>
-          <div className="flex h-full max-h-[60vh] w-full justify-center p-4">
+        <div className="flex h-[50vh] w-[90vw] flex-col rounded-lg bg-neutral-100 p-1 dark:bg-neutral-100-dark sm:w-[80vw] md:w-[70vw] md:p-6 lg:w-[60vw] xl:w-[50vw] 2xl:w-[40vw]">
```

Comment:
![image](https://github.com/ElektraInitiative/PermaplanT/assets/33787737/ea76f3c8-e534-47c1-bd6c-df77e3cd1348)
Consider making the height bigger again or responsive to viewports with smaller heights.
It gets pretty unusable when the window is half the height of my monitor.